package.json

```diff
@@ -9,6 +9,10 @@
         "prebuild": "rimraf dist",
         "build": "nest build",
         "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+        "prestart": "npm run typeorm migration:run",
+        "prestart:debug": "npm run typeorm migration:run",
```

Comment:
Migrations are executed before they are built. This menas you have to run the backend a second time to execute the migrations